benchmarks/test/my_post_query_cycle_script.py

```diff
@@ -0,0 +1,11 @@
+import sys
```

Comment:
This comment is for the two .py file names. Why did you add `my_` in front? I think it's better to remove it

Reply:
this folder is for testing and demoing.

Reply:
What does "my" stand for?